src/shared/components/pipeline-run-logs/PipelineRunLogs.tsx

```diff
@@ -141,7 +141,7 @@ class PipelineRunLogs extends React.Component<PipelineRunLogsProps, PipelineRunL
             <Nav onSelect={this.onNavSelect} theme="light">
               <NavList className="pipeline-run-logs__nav">
                 {taskRuns.map((task) => {
-                  const taskRun = obj.status.taskRuns[task];
+                  const taskRun = obj.status?.taskRuns?.[task];
```

Comment:
We shouldn't be using `pipelinerun.status.taskRuns` anymore, its deprecated.
```suggestion
                  const taskRun = taskRunFromYaml?.[task];
```